src/components/CashlinkSparkle.vue

```diff
@@ -0,0 +1,106 @@
rg-8" cx="363.32" cy="51.3" r="5.17" xlink:href="#mrg-5"/><radialGradient id="mrg-9" cx="124.32" cy="15.3" r="5.17" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#265dd7"/><stop offset="1" stop-color="#0582ca"/></radialGradient><radialGradient id="mrg-10" cx="29.82" cy="52.3" r="5.17" xlink:href="#mrg-9"/><radialGradient id="mrg-11" cx="255.32" cy="21.8" r="5.17" xlink:href="#mrg-9"/></defs><title>sparkle-mobile</title><g id="sparkle"><circle class="cls-1" cx="10.5" cy="39" r="2.5"/><circle class="cls-2" cx="8.5" cy="27.5" r="2.5"/><circle class="cls-1" cx="25" cy="26" r="2.5"/><circle class="cls-3" cx="22.5" cy="13.5" r="2.5"/><circle class="cls-2" cx="36.5" cy="8.5" r="2.5"/><circle class="cls-1" cx="53.5" cy="20" r="2.5"/><circle class="cls-4" cx="65" cy="40" r="2.5"/><circle class="cls-5" cx="92" cy="18" r="2.5"/><circle class="cls-5" cx="75" cy="22.5" r="2.5"/><circle class="cls-1" cx="84.5" cy="12" r="2.5"/><circle class="cls-5" cx="109.5" cy="17.5" r="2.5"/><circle class="cls-4" cx="127.5" cy="26" r="2.5"/><circle class="cls-5" cx="141.5" cy="10" r="2.5"/><circle class="cls-4" cx="167" cy="23" r="2.5"/><circle class="cls-5" cx="177" cy="12.5" r="2.5"/><circle class="cls-5" cx="191" cy="10" r="2.5"/><circle class="cls-4" cx="215" cy="22.5" r="2.5"/><circle class="cls-5" cx="218.93" cy="8.5" r="2.5"/><circle class="cls-4" cx="236.5" cy="27" r="2.5"/><circle class="cls-5" cx="249.5" cy="10" r="2.5"/><circle class="cls-4" cx="264.5" cy="26" r="2.5"/><circle class="cls-4" cx="278.5" cy="23" r="2.5"/><circle class="cls-5" cx="299.5" cy="13" r="2.5"/><circle class="cls-4" cx="303" cy="22" r="2.5"/><circle class="cls-1" cx="320" cy="8.5" r="2.5"/><circle class="cls-4" cx="324" cy="20.5" r="2.5"/><circle class="cls-5" cx="346.5" cy="19.5" r="2.5"/><circle class="cls-4" cx="341" cy="28.5" r="2.5"/><circle class="cls-6" cx="353" cy="8.5" r="2.5"/><circle class="cls-2" cx="364.5" cy="25.5" r="2.5"/><circle class="cls-1" cx="366.5" cy="41" r="2.5"/><circle class="cls-5" cx="354" cy="43.5" r="2.5"/><circle class="cls-4" cx="349" cy="52" r="2.5"/><circle class="cls-5" cx="366.5" cy="64" r="2.5"/><circle class="cls-4" cx="355.5" cy="81.5" r="2.5"/><circle class="cls-5" cx="19" cy="45" r="2.5"/><circle class="cls-5" cx="26.5" cy="38.5" r="2.5"/><circle class="cls-5" cx="33.5" cy="31.5" r="2.5"/><circle class="cls-4" cx="35.5" cy="44" r="2.5"/><circle class="cls-4" cx="8.5" cy="61.5" r="2.5"/><circle class="cls-4" cx="20" cy="68" r="2.5"/><circle class="cls-5" cx="8.5" cy="73" r="2.5"/><circle class="cls-7" cx="147" cy="21" r="2.5"/><circle class="cls-8" cx="288.5" cy="12.5" r="2.5"/><circle class="cls-9" cx="61.5" cy="11" r="2.5"/><circle class="cls-10" cx="206" cy="26" r="2.5"/><circle class="cls-11" cx="334" cy="14.5" r="2.5"/><circle class="cls-12" cx="361.5" cy="49.5" r="2.5"/><circle class="cls-13" cx="122.5" cy="13.5" r="2.5"/><circle class="cls-14" cx="28" cy="50.5" r="2.5"/><circle class="cls-15" cx="253.5" cy="20" r="2.5"/></g></svg>
```

Comment:
`v-if` would require a JS window-width listener, would it not?

Reply:
Yes, but worth it in my opinion.

Reply:
I think it's not worth it to introduce additional Javascript for a task just as well handled by less CSS. Since the unused DOM nodes are set to `display: none` I don't think they take up any processing power of the browser. And since this page that displays the CashlinkSparkle doesn't have anything else going on (also no animations or long lists), the additional (hidden) SVG nodes have no impact on browser performance whatsoever.